font-codegen/src/record.rs

```diff
@@ -40,6 +40,11 @@ pub(crate) fn generate(item: &Record) -> syn::Result<TokenStream> {
             impl FixedSize for #name {
                 const RAW_BYTE_LEN: usize = #( #inner_types::RAW_BYTE_LEN )+*;
             }
+
+            /// SAFETY: I am repr(c) + repr(packed), and all of my fields impl `JustBytes`.
```

Comment:
Not all readers will be Rusty enough to understand the full implications of this so despite being a nuisance I suggest spelling it out in a little more detail what all this means. For example, I would not have known repr(packed) => alignment 1 => can be converted from bytes at any offset without looking up the documentation.

Reply:
Also iiuc the codegen is what is actually ensuring the things asserted here. The trait is merely a marker we claim we will only put on things that are so structured. To me that's a bit different than what the comment says.

Reply:
This comment is really just about satisfying #435; I think a better solution is to centralize all the docs around how this works in the trait, and then I'll just link to the trait from the generated code.